### ISSUE TITLE
Writer: Formatting marks not working

### ISSUE THREAD

**Issue by pedropintosilva** [CollaboraOnline/online#7423]
# Description

View tab > Formatting Marks

Mike D. during the COOL Weekly Meeting
    I turned it on, but no marks appeared.

I have tested and the same odd behaviour is possible to be replicated when:

Home > Format marks
![image](https://github.com/CollaboraOnline/online/assets/65948705/7d06b7d1-537e-4db8-92ea-bcc05bea13b1)

It seems clicking either button works only once after that we get cropped marks (page limiter/border is cropped) and then we never get the paragraph marks etc again. Sometimes we do get it again and then nothing.

https://github.com/CollaboraOnline/online/assets/65948705/d8d130d3-5953-4507-8fd5-269e7cfa337f

**Note: I get consistently `Error in parsing value for 'top'.  Declaration dropped. when pressing the button`**

# Logs

## Using the button from Home tab

web dev tools console messages: 
[console-export-2023-10-12_14-10-56.txt](https://github.com/CollaboraOnline/online/files/12882716/console-export-2023-10-12_14-10-56.txt)

## Using the button from View tab

Web dev tools console messages: 
[console-export-2023-10-12_14-14-40.txt](https://github.com/CollaboraOnline/online/files/12882718/console-export-2023-10-12_14-14-40.txt)

# Version
COOLWSD version: 23.05.5.1 (git hash: dffef0e9 (E))
LOKit version: Collabora Office 23.05.5.1 (git hash: ece27b5)
Served by: openSUSE Leap 15.5
Server ID: ea7a1b48

**Comment by Tex2002ans** [comment]
I was doing:

- Go to "View" Tab
- Press "View Formatting" button.

Even though it toggled on, I wasn't even able to see formatting marks at all.

So maybe those 2 buttons also act slightly differently?

- - -

In Staging this morning:

> COOLWSD version: 23.05.5.1 (git hash: dffef0e9 (E))
> LOKit version: Collabora Office 23.05.5.1 (git hash: 9f03693)
> Served by: openSUSE Leap 15.5
> Server ID: 3a813eff

I pressed, nothing happened.

- - -

In recent build (as of last week):

> COOLWSD version: 23.05.4.2 (git hash: 0ba51f81)
> LOKit version: Collabora Office 23.05.4.2 (git hash: b5a7948)
> Served by: Linux Mint 21.1
> Server ID: 78797e84

It just acts weird. It turns on/off, but feels like it sometimes gets "stuck". (It partially worked, but then later clicks of button don't toggle.)

I didn't see any errors in console while this happens.

**Comment by timur-g** [comment]
Seems that 23.05.0.5 was OK, then in .2 hash: 91c166d NOK after click marks cannot be turned off, ithen n .4 hash: 938b8e3 was OK, in .5 hash: a1b2091a NOK turn on and off work but not more. Could not verify or bisect at the moment.

**Comment by timur-g** [comment]
After document is loaded (regardless if Dark or Light mode), formatting marks can be turned on and not off.
But, if then mode is changed (Dark-Light), then both on/off work fine.

**Comment by timur-g** [comment]
Regression from:
commit 9d6b1d148bfc0ab055c72a8b210bfd95fcc51bea
Author: Rash419 <rashesh.padia@collabora.com>
Date:   Thu Sep 28 16:44:45 2023 +0530
    don't request new tiles unnecessarily on canonicalidchange

@Rash419 please see.

**Comment by Tex2002ans** [comment]
I tested the button on staging server:

> COOLWSD version: 23.05.6.4 (git hash: b1f0834a (E))
> LOKit version: Collabora Office 23.05.6.4 (git hash: 21abead)
> Served by: openSUSE Leap 15.5
> Server ID: 4e3ac136

and it's still really busted.

- - -

Also following @timur-g 's idea...

A1. Go to "View" tab.
A2. Press "Formatting Marks" button.
   - Nothing happens.
   - No formatting marks appear.

or:

B1. Go to "View" tab.
B2. Press "Dark Mode".
B3. Press "Formatting Marks" button.
   - Page flips to light, while UI/surroundings stay dark.
      - Page seems to "get stuck" in light mode, no matter how far I scroll or how many times I toggle "Dark Mode" button.
   - No formatting marks appear.

I tried both methods after completely reloading/refreshing the tab.
